src/index.html

```diff
@@ -17,16 +17,27 @@
     />
```

Comment:
The `link` tag and its attributes are not formatted correctly. If the HTML-element has long attribute values or number of attributes is more than 2 - start each one, including the first, on the new line with 2-space indentation related to tag. Tag’s closing bracket should be on the same level as opening one. [Keep your attributes correctly formatted]